server/cp/macros/update_translation_metadata_macro.py

```diff
@@ -14,6 +14,14 @@
 
 logger = logging.getLogger(__name__)
 
+fields = ["Presse Canadienne staff", "L'Associated Press", "Canadian Press Staff", "The Associated Press"]
```

Comment:
it would be better to use `qcode` instead of names, so if names are updated the macro won't have to be modified.